packages/shared-ui/src/elements/welcome-panel/project-listing.ts

```diff
@@ -447,6 +447,24 @@ export class ProjectListing extends LitElement {
 
     this.addEventListener("click", this.#hideBoardServerOverflowMenuBound);
 
+    for (const boardServer of this.boardServers) {
+      const closuredName = boardServer.name;
+      boardServer.addEventListener(
+        GraphBoardServerRefreshEvent.eventName,
+        () => {
+          // Listen to all, react only to the current.
+          if (closuredName == this.selectedBoardServer) {
```

Comment:
I don't think we need this, right?

Reply:
we do, otherwise how that event is going to be propagated? (I tried removing it and it didn't refresh)